int8 INACTIVE    = 0
int8 ACTIVE      = 1
int8 PAUSED      = 2
int8 PURGED      = 3
int8 SUCCEEDED   = 4
int8 ERROR       = 5

time stamp
string name
int8 status
MetricResult user_result
